spec/system/placements/organisations/support_user_selects_an_organisation_type_to_add_spec.rb

```diff
@@ -0,0 +1,75 @@
+require "rails_helper"
+
+RSpec.describe "Placements / Schools / Support User adds a Provider",
+               type: :system do
+  before do
+    given_i_sign_in_as_colin
+    when_i_click_add_organisation
+  end
+  after { Capybara.app_host = nil }
+
+  scenario "Colin selects to add an ITT provider" do
+    when_i_select_the_radio_option("ITT provider")
+    and_i_click_continue
+    then_i_see_the_title("Enter a provider name, UKPRN, URN or postcode")
+  end
+
+  scenario "Colin selects to add a school" do
+    when_i_select_the_radio_option("School")
+    and_i_click_continue
+    then_i_see_the_title("Enter a school name, URN or postcode")
+  end
+
+  scenario "Colin doesn't select an organisation type" do
+    and_i_click_continue
+    then_i_see_an_error("Select an organisation type")
```

Comment:
We should see the error twice. Once in the error summary and once above the input select. Good to check those are both here.

Reply:
resolved in [Rspec for School decorator](https://github.com/DFE-Digital/itt-mentor-services/pull/73/commits/791089c78da384460beb9f1dedd3496e02c1e721)